packages/lib-react-components/src/Media/Media.stories.js

```diff
@@ -1,15 +1,21 @@
-import zooTheme from '@zooniverse/grommet-theme'
-import { Grommet, Box, Text } from 'grommet'
+import { Box, Text } from 'grommet'
 
 import Media from './Media'
 import ZooniverseLogo from '../ZooniverseLogo'
 
+/** This will not work with @storybook/testing-react composeStory function */
+// import readme from './README.md'
```

Comment:
❗ As discussed during the FE dev standup last Thursday, _sometimes_ we can't import READMEs for the Story. In this example, this is because the imported README.md would bork `Media.spec.js`, which uses `composeStory` to use the Storybook Story as part of its React Testing Library tests.

While this is unfortunate, it's better to have Readmes hidden from Storybook than to have broken tests, and we can always revisit a solution for this later.

Note 1: other components, e.g. AdminCheckbox, don't import their Stories into the RTL tests, so they can keep their READMEs.

Note 2: I'm very much in favour of leaving the code here and commenting them out, _instead of just deleting the non-functioning code._ It's important to know _why_ it doesn't work.

Reply:
[note to self] While reading up on composeStory, I'm seeing [notes](https://storybook.js.org/addons/@storybook/testing-react) from `@storybook/testing-react` detailing some changes to composeStory, and we can change `import { composeStory } from '@storybook/testing-react'` to `import { composeStory } from '@storybook/react`. Could that fix the Readme imports? Maybe, but the caveat is this only works in Storybook 7+, and lib-react-components is still on Storybook 6.

Reply:
Thanks Shaun! I'll save this note as something to try when manually upgrading to Storybook v7.